lib/siwapp/invoices/invoice.ex

```diff
@@ -174,10 +174,9 @@ defmodule Siwapp.Invoices.Invoice do
       get_field(changeset, :draft) ->
         changeset
 
-      is_nil(get_change(changeset, :series_id)) ->
-        changeset
-
-      is_nil(get_change(changeset, :number)) ->
+      # Next number is assigned when changing serie and no number if provided or draft status is removed
```

Comment:
This phrase is very confusing. Try to say it in another way. At least I can't understand what you mean here.